src/sentry/static/sentry/app/views/organizationGroupDetails/actions.jsx

```diff
@@ -144,14 +144,12 @@ const GroupDetailsActions = createReactClass({
     const discoverQuery = {
       id: undefined,
       name: group.title || group.type,
-      fields: ['title', 'url', 'count(id)', 'project', 'last_seen'],
-      widths: [400, 200, -1, 200, -1],
-      orderby: '-count_id',
+      fields: ['title', 'event.type', 'project', 'user', 'timestamp'],
```

Comment:
If this is an issue won't the event.type be fairly redundant? Same with `project` it would always have the same value. Could we include something like the environment, release, level or logger?

Reply:
Yea that's a good point. I'll clarify if we can drop these columns or replace them with other fields with Bowen.

Reply:
That makes sense. My suggestion to keep it consistent with 'All Events' was so that we weren't 'enabling' Business-features (custom columns) for users on our Team plan.

Thinking about it more, that's not strong enough of a reason to show redundant info here to our Business users. Ideally we split it between Team or Business, but I don't think that's worth the time/effort investment currently.

Let's drop project and event.type and add in `release` and `environment`. Reload metrics indicate those are some of the most common columns users add to Discover queries as well.